tracker/tracker-assist/src/Assist.ts

```diff
@@ -243,21 +268,43 @@ export default class Assist {
     if (this.options.config) {
       peerOptions['config'] = this.options.config
     }
+    const lStream = {}
     const peer = this.peer = new Peer(peerID, peerOptions)
-    // app.debug.log('Peer created: ', peer)
     // @ts-ignore
     peer.on('error', e => app.debug.warn('Peer error: ', e.type, e))
     peer.on('disconnected', () => peer.reconnect())
-    peer.on('call', (call) => {
-      app.debug.log('Call: ', call)    
-      if (this.callingState !== CallingState.False) {
+
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
+    peer.on('call', async (call) => {
+      const onCallEnd = this.options.onCallStart()
+      const handleCallEnd = () => {
```

Comment:
that is now re-created on each call connection. 
It should be in the upper common context; all the `call` objects should belong to  corresponding calling Agents.